falco-exporter/values.yaml

```diff
@@ -34,6 +34,16 @@ serviceAccount:
   # Annotations to add to the service account
   annotations: {}
   # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template.
+  # If set and create is false, an already existing serviceAccount must be provided.
+  name:
+
+podSecurityPolicy:
+  # Specifies whether a PSP, Role and RoleBinding should be created
+  create: true
```

Comment:
Since the feature needs `apiVersion: policy/v1beta1`, I would not enable it by default.
It is also not enabled in other charts of this repo.

Reply:
Furthermore, it worth to add a capabilities check in the template too, for example:
```
{{- if and .Values.podSecurityPolicy.create (.Capabilities.APIVersions.Has "policy/v1beta1") }}
```
WDYT?

Reply:
I'll add this additional check.

Reply:
.. and disable the PSP stuff be default :smile: